charts/consul/values.yaml

```diff
@@ -105,7 +105,7 @@ global:
   # image that is used for functionality such as catalog sync.
   # This can be overridden per component.
   # @default: hashicorp/consul-k8s-control-plane:<latest version>
-  imageK8S: "hashicorp/consul-k8s-control-plane:0.35.0"
+  imageK8S: "hashicorp/consul-k8s-control-plane:0.35.1"
```

Comment:
This should be 0.36. I think there are too many improvements in the changelog for a patch release.

Reply:
Cool, this was a bad assumption by me. I had mapped `0.35.0` to beta 1 so I figured `0.35.1` would be beta 1.